Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -109,7 +109,28 @@ class SentryUserFeedbackForm: UIViewController {
     }
     
     func submitFeedbackButtonTapped() {
-        // TODO: validate and package entries
+        var missing = [String]()
+        
+        if config.formConfig.isNameRequired && fullNameTextField.hasText {
+            missing.append("name")
+        }
+        
+        if config.formConfig.isEmailRequired && emailTextField.hasText {
```

Comment:
`m`: Still in progess, I know, but I think it should be like that:
```suggestion
        if config.formConfig.isNameRequired && !fullNameTextField.hasText {
            missing.append("name")
        }
        
        if config.formConfig.isEmailRequired && !emailTextField.hasText {
```

Reply:
Oops, you're right, I changed these from `.text.isEmpty` without adding negation 🤦🏻